service/src/main/java/org/kiwiproject/champagne/resource/filter/DeployableSystemRequestFilter.java

```diff
@@ -0,0 +1,33 @@
+package org.kiwiproject.champagne.resource.filter;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kiwiproject.champagne.dao.DeployableSystemDao;
+import org.kiwiproject.champagne.model.DeployableSystemThreadLocal;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+
+public class DeployableSystemRequestFilter implements ContainerRequestFilter {
+
+    private static final String DEPLOYABLE_SYSTEM_HEADER_NAME = "DeployableSystem";
```

Comment:
Do we need to prefix this with "X-" since it's a custom header? Or is that no longer "a thing" when creating custom headers? Or, is there some other naming convention that is now preferred?

Reply:
Apparently as of 2012, it is recommended to use sensible names instead of prefixing with "X-".  Here is a good writeup of the thought process:

https://stackoverflow.com/a/3561399

That being said, not sure "DeployableSystem" is **sensible**